specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/stable/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,951 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PhoneNumbersClient",
+    "description": "The phone numbers client uses the Communication Services to acquire and manage phone numbers.",
+    "contact": {
```

Comment:
Let's remove the "contact" section. It is not needed anymore and will not be a monitored address